charts/operator-wandb/charts/nginx/templates/configmap.yaml

```
@@ -0,0 +1,37 @@
+apiVersion: v1
+kind: ConfigMap
+metadata:
+  name: {{ include "nginx.fullname" . }}
+  labels:
+    {{- include "nginx.commonLabels" . | nindent 4 }}
+    {{- include "nginx.labels" . | nindent 4 }}
+    {{- if .Values.configMap.labels -}}
+    {{-   toYaml .Values.configMap.labels | nindent 4 }}
+    {{- end }}
+  annotations:
+    {{- if .Values.configMap.annotations -}}
+    {{-   toYaml .Values.configMap.annotations | nindent 4 }}
+    {{- end }}
+data:
+  nginx.conf: |
+    worker_processes  auto;
+
+    error_log  /var/log/nginx/error.log notice;
+    pid        /tmp/nginx.pid;
+
+
+    events {
+        worker_connections  1024;
+    }
+
+    http {
+      server {
+        listen 8080;
+        location / {
+          proxy_pass http://{{ .Release.Name }}-app.default.svc.cluster.local:8080;
+        }
+        location /console {
+          proxy_pass http://{{ .Release.Name }}-console.default.svc.cluster.local:8082;
```

```suggestion
          proxy_pass http://{{ .Release.Name }}-console.svc.cluster.local:8082;
```

This assumes default namespace? But it might not be in default? I think we can just remove it.